pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -435,6 +435,8 @@ type NodeGroup struct {
 	VolumeEncrypted *bool `json:"volumeEncrypted,omitempty"`
 	// +optional
 	VolumeKmsKeyID *string `json:"volumeKmsKeyID,omitempty"`
+	// +optional
+	VolumeIOPS *int `json:"volumeIOPS"`
```

Comment:
The naming is a bit consistent from `volumeKmsKeyID`. The Go convention is to use all uppercase for acronyms but the AWS API and docs use `KmsKeyID` and `Iops` but I believe we should prefer the Go convention here.

Reply:
We will have to fix `volumeKmsKeyID` in the next version of the API. I think we will need to move all the volume params into a struct anyway, as there are quire a few now, and we will need to review if supporting extra volumes could be of use.

Reply:
I've opened #1019